WordPress/Classes/ViewRelated/Notifications/Comment Moderation/CommentModerationOptionsView.swift

```diff
@@ -151,12 +151,24 @@ private extension CommentModerationOptionsView.Option {
     }
 }
 
-final class CommentModerationOptionsViewController: BottomSheetContentViewController {
-
-    typealias Option = CommentModerationOptionsView.Option
+private extension CommentModerationState {
+    init(option: CommentModerationOptionsView.Option) {
+        switch option {
+        case .approve:
+            self = .approved(liked: false)
```

Comment:
[Discussion] The initial valued of `liked` should be fetched from the `comment`object. Perhaps you were planning to address this in a separate PR?

Reply:
Resolved in [758bd35](https://github.com/wordpress-mobile/WordPress-iOS/pull/23225/commits/758bd35ea8d4846fe451850a38e5b07fe3d964c3)